classes/Combination.php

```diff
@@ -529,15 +535,14 @@ public function getColorsAttributes()
      *
      * @param int $idProductAttribute
      *
-     * @return float mixed
+     * @return float
      *
      * @since 1.5.0
      */
     public static function getPrice($idProductAttribute)
     {
-        return Db::getInstance(_PS_USE_SQL_SLAVE_)->getValue(
-            '
-			SELECT product_attribute_shop.`price`
+        return (float) Db::getInstance(_PS_USE_SQL_SLAVE_)->getValue(
```

Comment:
I'm not sure we want to cast this into float (lack of precision and all..) Maybe it's safer to return a string, even if the DB type is Decimal there might be a chance that we lose precision when the DB output is cast to a PHP float?